README.md

```diff
@@ -1,6 +1,63 @@
 # [ng-suggestion](http://ianwalter.github.io/ng-suggestion/)
 *Flexible AngularJS typeahead / autocomplete / suggestion / predictive search directive*
 
-I don't have time for docs right now just read the source.
-
+A wise man once said, *"I don't have time for docs right now just read the source."*  
```

Comment:
Nice, but this should be removed when we get the README to a more complete state